public/locales/en/expectations.json

```diff
@@ -29,7 +29,17 @@
       "item-2": "for official government business (special or diplomatic passports)"
     }
   },
-  "description-privacy": "Have you read and agree to the privacy and terms of service?",
+  "description-privacy": {
+    "1": "The information provided is collected under section 5.1(1)(a)(i) of the Department of Employment and Social Development Act and/or section 12(e) of the Canadian Passport Order for the purpose of providing you with the status of your passport application. Refusing to provide information will result in not being able to view the status of your passport application.",
+    "2": "In addition, we collect your email address only if you require Employment and Social Development Canada (ESDC) to send your passport application file number by email. Without your email address, ESDC cannot send your passport application number electronically.",
+    "3": "Your personal information may be shared within Service Canada and with Immigration, Refugees and Citizenship Canada to improve services through evaluations, reports and research. This will never result in an administrative decision being made about you.",
+    "4": "ESDC carries out this status checker application using services from third-party provider Microsoft Corporation, which may be subject to foreign laws. For more information, see <2>Microsoft Data Privacy Policy & Privacy Principles</2>.",
```

Comment:
Change `<2>Microsoft Data Privacy Policy & Privacy Principles</2>` to `<Link>Microsoft Data Privacy Policy & Privacy Principles</Link>` and use `Trans` property `components` like this:

```tsx
<Trans
  i18nKey={'expectations:description-privacy.4'}
  components={{ Link: <ExternalLink href={t('description-privacy.4-link')} /> }}
/>
```

https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0

Do this of the other strings.

Reply:
A child element is needed for ExternalLink so a dummy "link" was added